framework/src/media/media_player.c

```diff
@@ -323,7 +321,7 @@ int player_worker(void *args)
 	g_pc.pcmout = NULL;
 
 	PLAY_LOCK();
-	g_pc.state = PLAY_STOPPING;
+	g_pc.state = PLAY_ERROR;
```

Comment:
Why ERROR?

Reply:
Later i will edit g_pc.state whole. and if state is not ERROR, restart is not possible.

Reply:
Same question. Why ERROR? Do you think starting from ERROR state is correct? Not IDLE? Not STOPPED?